api/python/provisioner/utils.py

```diff
@@ -186,7 +186,8 @@ def run_subprocess_cmd(cmd, **kwargs):
         logger.exception(f"Failed to run cmd '{cmd}'")
         raise SubprocessCmdError(cmd, _kwargs, exc) from exc
     else:
-        logger.debug(f"Subprocess command resulted in: {res}")
+        logger.debug("""Subprocess command {} resulted in - stdout: {},returncode: {},stderr: {}""".format(
+            res.args, res.stdout, res.returncode, res.stderr))
```

Comment:
no triple quotes please and f-string is proffered over older `format`, e.g.
```
logger.debug(
    f"Subprocess command {res.args} resulted in - stdout: {res.stdout}, "
    f"returncode: {res.returncode},stderr: {res.stderr}"
)
```

Reply:
Sure.